setup.py

```diff
@@ -266,9 +266,8 @@ def build_dir():
         "packages": ['khmer', 'khmer.tests', 'oxli', 'khmer._oxli'],
         "package_data": {'khmer/_oxli': ['*.pxd']},
         "package_dir": {'khmer.tests': 'tests'},
-        "install_requires": ['screed >= 1.0', 'bz2file'],
-        "setup_requires": ["pytest-runner>=2.0,<3dev", "setuptools>=18.0",
-                           "Cython>=0.25.2"],
+        "install_requires": ['screed >= 1.0', 'bz2file', 'Cython>=0.25.2'],
```

Comment:
I don't think we're distributing `.cpp` files now. I'm not against that, but honestly I'm mostly concerned about getting the khmer and kevlar docs to build again. I'd suggest (barring any YUGE concerns) that we merge this and then open up a thread to discuss The Right Way to do it.